utils/staking-miner/src/dry_run.rs

```diff
@@ -64,9 +64,10 @@ async fn print_info<T: EPM::Config>(
 		let deposit = EPM::Pallet::<T>::deposit_for(&raw_solution, snapshot_size);
 		log::info!(
 			target: LOG_TARGET,
-			"solution score {:?} / deposit {:?}",
+			"solution score {:?} / deposit {:?} / length {:?}",
 			&raw_solution.score.iter().map(|x| Token::from(*x)).collect::<Vec<_>>(),
 			Token::from(deposit),
+			raw_solution.encode().len(),
```

Comment:
This seems quite expensive just to do a log message - is there somewhere else this is already encoded and we can just pul the length from that? Or maybe we can use `size_hint(&self) -> usize`?

Reply:
yeah it could, but it would make the code look less ergonomic and since this is dry-run and not at all time-sensitive, I intentionally overlooked it.